docs/en/wallet-attestation.rst

```diff
@@ -478,7 +478,66 @@ The body of the Wallet Attestation JWT MUST contain:
       -
     * - **presentation_definition_uri_supported**
       - Boolean value specifying whether the Wallet Instance supports the transfer of presentation_definition by reference. MUST be set to false.
-      -
+
+Wallet Instance Lifecycle
+------------
+
+The ability of the Wallet Instance to obtain a Wallet Attestation is bound to its current state. 
+The Wallet Instance calculates its current state based on its local credentials storage and its revocation state on the Wallet Provider's backend, if present.
+
+The Wallet Instance lifecycle defines all possible states a Wallet Instance can be in, as well as the transitions between them. The Wallet Instance lifecycle is illustrated in the following diagram:
+
+.. figure:: ../../images/wallet_instance_lifecycle.svg
+   :name: Wallet Instance Lifecycle
+   :alt: The image illustrates the Wallet Instance lifecycle, with the states explained below.
+   :target: https://www.plantuml.com/plantuml/uml/SoWkIImgAStDuOhMYbNGrRLJyCm32kNafAPOAMH2c5mAG00N1YloBqWjIYp9pCzBpB5IA4ijoaoh1Ab25WUh2qlCoKm1gW1HYIMf83KGCKnJClDmg799JKmkoIm3IW1DAaejoyzEHRSBfpfCbmEzQQLGceVaDOH6x4emxS9KWd0mfgH3QbuAC801
+
+
+A Wallet Instance can obtain a Wallet Attestation if it's in either `Installed`, `Operational` or `Valid` state; that implies that a `Deactivated` Wallet Instance cannot obtain a Wallet Attestation hence it cannot interact with other entities such as PID/(Q)EAA Providers and Relying Parties.
```

Comment:
This is still not clear. the point is that the Installed, Operational and Valid states depend on a valid Wallet Attestation. This means that if, during the issuance of a Wallet Attestation, something goes wrong and the Wallet Attestation cannot be provided, the state of the Wallet must change. At the same time, a Wallet Instance in a Deactivated state MUST not obtain a valid Wallet Attestation (e.g. this could be the case of a Wallet Instance revocation triggered by the User). Am I missing something?